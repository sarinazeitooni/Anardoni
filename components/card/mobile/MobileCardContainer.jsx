import React from 'react';
import style from './scss/mobile-card.module.scss';
import messages from "./messages/messages";
function MobileCardContainer({title,subtitle,containerImage}){
    return(
        <div className={style['mobile-card-container']}>
            <div className={style['titles-container']}>
                <div className={style['titles']}>
                    <div className={style['title']}>{title}</div>
                    <div className={style['subtitle']}>{subtitle}</div>
                </div>
                <div className={style['get-button']}>{messages.getButton}</div>
            </div>
            <div className={style['main-image']} style={{background :  `url("${containerImage}") no-repeat center center / cover`}}>
            </div>
        </div>
    )
};
export default MobileCardContainer;