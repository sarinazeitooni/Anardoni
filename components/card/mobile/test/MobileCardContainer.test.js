import React from 'react';
import { shallow } from 'enzyme';
import MobileCardContainer from "../MobileCardContainer";
describe("MobileCardContainer component" , ()=>{
    const container = shallow(<MobileCardContainer/>);
    it("it should render a div",()=>{
        expect(container).toMatchSnapshot();
    });
    it("it should render a mobile-card-container" ,()=>{
        expect(container.find('div.mobile-card-container')).toBeTruthy();
    });
});