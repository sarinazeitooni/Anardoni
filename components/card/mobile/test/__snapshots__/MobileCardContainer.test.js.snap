// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileCardContainer component it should render a div 1`] = `ShallowWrapper {}`;
